controllers/selfnoderemediation_controller.go

```diff
@@ -624,3 +630,28 @@ func (r *SelfNodeRemediationReconciler) restoreNode(nodeToRestore *v1.Node) (ctr
 	// all done, stop reconciling
 	return ctrl.Result{Requeue: true}, nil
 }
+
+func (r *SelfNodeRemediationReconciler) updateSnrStatusLastError(snr *v1alpha1.SelfNodeRemediation, err error) error {
+	var lastErrorVal string
+
+	if err != nil {
+		lastErrorVal = err.Error()
+	} else {
+		lastErrorVal = "successful reconcile"
```

Comment:
I think setting `lastErrorVal` to "successful reconcile" is a bit confusing.
I'd suggest something like:
```
If err == nil {
  return clearSnrStatusLastError()
 }
```